modules/apim-adapter/src/main/java/com/axway/apim/api/model/apps/ClientApplication.java

```diff
@@ -148,13 +157,14 @@ public boolean equals(Object other) {
 		if(other == null) return false;
 		if(other instanceof ClientApplication) {
 			ClientApplication otherApp = (ClientApplication)other;
+			Comparator c = Comparator.comparing(ClientAppCredential::getCredentialType).thenComparing(ClientAppCredential::getId);
```

Comment:
I have to sort the collections in order to determine if they are equals

Reply:
Nice! Looks like I have to learn Java >8 stuff 😄